README.md

```diff
@@ -19,7 +25,7 @@ meet in the same [Zoom channel](https://zoom.us/j/361123509) as is used for SIG-
 ## Working Items
 
 * Multi-Network CRD Specification
-  * [Repositories](https://github.com/K8sNetworkPlumbingWG/multi-net-spec): It contains finalized version.
+  * [Repositories](https://github.com/k8snetworkplumbingwg/community/issues): It contains finalized version.
```

Comment:
Not sure this change was on purpose. Were you jsut trying to change the case of the `/K8sNetworkPlumbingWG/`?

Reply:
Thanks! Copy/paste error, thanks for catching it. Yeah, intended to match the changed case (from last year!)